causal_testing/testing/causal_test_case.py

```diff
@@ -56,7 +57,7 @@ def get_treatment_variables(self):
 
     def get_outcome_variables(self):
         """Return a list of the outcome variables (as strings) for this causal test case."""
-        return [v.name for v in self.outcome_variables]
+        return [self.outcome_variable.name]
```

Comment:
Does this really need to be a list if it's only ever going to contain one element? I know we want to keep in mind that we eventually want to support multiple treatments but this seems a little illogical...

Reply:
I completely agree, I started refactoring this and I think it will make the code in many places much neater removing the dictionary configuartions. However looking at it the scope of it definitely deserves it's own PR so i've moved it onto a separate branch and will make a new PR in time :)